pkg/cluster/cache.go

```diff
@@ -117,12 +113,12 @@ var HostCluster GetHostClusterFunc = GetHostCluster
 // GetHostCluster returns the kube client for the host cluster from the cache of the clusters
 // and info if such a client exists
 func GetHostCluster() (*CachedToolchainCluster, bool) {
```

Comment:
Same here. Shouldn't we have a single `GetClusters()` function instead of two `GetHostCluster()` and `GetMemberClusters()`?
Also we need to update the description of the function and stay why a single function is enough (there is an assumption that in the host namespace there are member clusters only and in the member namespace there is a single host cluster only).

Reply:
We do require single host cluster to be fetched out for few of the scenarios in member operator . If i make a single func instead of two over here, this same gethostcluster() logic will have to be added in member operator. so the only difference here would be that we transfer the logic from toolchain-common to member-operator. 
So do we remove the dependency of this logic in toolchain-common and add it to member-operator.?

Reply:
I'm leaning towards @alexeykazakov's proposal here to just have a single `GetClusters` in `toolchain-common`. We can have `GetHostCluster` function in the `member-operator` that would just use `cache.GetClusters()` and check that there is exactly 1 result and return that.